rfcs/52-unstable_features.md

```diff
@@ -0,0 +1,65 @@
+# Feature Name: `unstable_features`
+
+## Summary
+
+Accepting PRs for new features that may not have reach the high bar we set for Bevy, and disable their changes with a feature gate.
+
+## Motivation
+
+Some features would benefit from iterative work in Bevy, but that may be hard to do when a PRs need a high level of quality to be merged in.
+
+In this RFC, "feature" means the new feature or change under development, and "feature gate" the conditional compilation flag. "unstable" means a feature that is subject to change, not something that will break Bevy.
+
+## User-facing explanation
+
+So you want to contribute a new feature to Bevy! You have a few paths available to you, depending on the scope of the feature:
+- A small feature with a clear implementation? Go ahead and open that PR!
+- A large feature with an open design space that could use some discution about how to implement it? A RFC would be good.
+- Something in between that would benefit from iterative work? Here come the unstable feature workflow for you!
```

Comment:
I think large RFC'ed changes should also be subject to this workflow too. It'd also help isolate the effects of implementing individual parts of the larger design before it becomes stabilized.

Reply:
Yes I agree, but in those case I think the workflow should be RFC first, then unstable feature once the RFC is more or less agreed on. I tried to mention that in the "implementation" part, would that work for you?

Reply:
I see it now, but it's not particularly focused upon. Could you expand on each potential workflow down below?

Reply:
I tried!